src/simple_tracker_interfaces/msg/TrackPoint.msg

```diff
@@ -0,0 +1,3 @@
+geometry_msgs/Pose2D center
+int32 state
```

Comment:
What is stored in this state? This is pretty vague.

Reply:
Is there a frame timestamp associated with a track point?

Reply:
So the state is essentially the state of the track, which we currently have 3 options for: PROVISIONARY_TARGET = 1, ACTIVE_TARGET = 2, LOST_TARGET = 3. This allows me to draw the trajectory and bounding box as 1 = yellow, 2 = green, 3 = red. So this could probably be renamed to tracking_state or something like that to be more explicit.

Reply:
Please help me understand how that makes sense.

Each "track point" is a measurement right? Doesn't that imply that each point should have an associated timestamp? Otherwise it will be impossible to work out time-dependent analysis including triangulation and eventually full state estimates including velocity.

Reply:
Okay, I see what you are asking, we don’t currently record or store that level of detail for each track point.